HPXMLtoOpenStudio/measure.rb

```diff
@@ -168,12 +168,15 @@ def self.create(hpxml, runner, model, hpxml_path, epw_path, weather, output_dir,
 
     # Init
     check_file_references(hpxml_path)
+    # Fixme: do we need to check system number of speed?
```

Comment:
the schedule modification happens at the top of the measure, and number of speeds is an optional user input so we don't guarantee to have that info at this point, that's why I put a fixme here.